packages/opentelemetry-sdk-trace-base/src/Tracer.ts

```diff
@@ -117,6 +117,10 @@ export class Tracer implements api.Tracer {
       links
     );
 
+    // according to the spec an empty Tracestate means it should be cleared but
```

Comment:
Would it make sense to handle null given the same can be done by passing the result of `createTraceState` ? It's shorter but I don't remember seeing null anywhere else in the API. Either way agreed the comment isn't great, and the behaviour should probably also be described in the new field's doc comment.

Reply:
I don't think we should add special handling for `null`. I used `== null` above which includes `null` and `undefined` like the `??` operator used in your change.

If a sampler want's to clear the TraceState it should return an empty instance by calling `api.createTraceState()`.
But I would expect that a sample is usually not clearing the whole TraceState as it may contain arbitrary keys not related to sampling. Instead I assume they only set/clear/update sampler specific keys like the `ot` key used by [consistent probability samplers](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.16.0/specification/trace/tracestate-probability-sampling.md#consistent-probability-sampling).

Reply:
Gotcha, so the behaviour should already be correct. I updated the comments.